src/packages/core/src/server.ts

```diff
@@ -136,6 +137,10 @@ export default class Server {
 
     const initializePromise = this.initialize();
 
+    // necessary for `Promise.allSettled` to be shimmed
+    // in `node@10`
+    allSettled.shim();
```

Comment:
I don't think we should `shim` it everywhere; let's just use `allSettled([...])` directly.

Reply:
Using `allSettled([...])` directly with the typescript version you're using gives this error in several tests: https://github.com/es-shims/Promise.allSettled/issues/5

Reply:
Can we just ts-ignore it away for now? Upgrading breaks many other types.